tools/polarshift.rb

```diff
@@ -497,14 +497,19 @@ def clone_run(test_run_id: nil, options: nil)
       template_hash = {}
       case_status = options.status
       run_title = options.title
-      valid_statuses = ['Passed', 'Failed', 'Waiting', 'Running', 'Blocked']
+      # 'All' statuss mean we ignore the status and just filter on subteam name
+      valid_statuses = ['Passed', 'Failed', 'Waiting', 'Running', 'Blocked', 'All']
```

Comment:
`All` does not sounds like a valid case status.

Perhaps we can try, when user does not specify case status, then we select test cases that with a status in any of that in valid_statuses, which means `all`, but should not introduce a new invalid status `All`.

Reply:
@liangxia ok, now if nothing is given then it will return all.  PTAL